separate_instance/backend/Dockerfile

```diff
@@ -0,0 +1,109 @@
+##################################################################################
```

Comment:
what is the change in this file?

Reply:
...
ENV PORT="4002"

# Labels
LABEL org.label-schema.name="gradido:backend2"
LABEL org.label-schema.description="Gradido GraphQL Backend 2"
...
LABEL org.label-schema.vendor="Gradido Community 2"
...